definitions/infra-container/definition.yml

```diff
@@ -26,5 +26,3 @@ goldenTags:
 compositeMetrics:
   goldenMetrics:
   - golden_metrics.yml
-  summaryMetrics:
```

Comment:
Might be worth adding a comment here explaining why we're preventing them from being (re-)added until we fix